packages/terra-clinical-action-header/src/ActionHeader.jsx

```diff
@@ -100,13 +103,13 @@ const ActionHeader = ({
   const previousText = intl.formatMessage({ id: 'Terra.Clinical.ActionHeader.previous' });
   const nextText = intl.formatMessage({ id: 'Terra.Clinical.ActionHeader.next' });
 
-  const closeButton = onClose ? <Button icon={<IconClose ariaLabel={closeText} />} onClick={onClose} /> : null;
-  const backButton = onBack ? <Button icon={<IconLeft ariaLabel={backText} />} onClick={onBack} /> : null;
+  const closeButton = onClose ? <Button icon={<IconClose />} aria-label={closeText} onClick={onClose} /> : null;
+  const backButton = onBack ? <Button icon={<IconLeft />} aria-label={backText} onClick={onBack} /> : null;
```

Comment:
It should be on the button element, I believe screen readers have trouble announcing aria-label when its in the span or svg inside the button.